### ISSUE TITLE
Feature request: Notify recipient of new transaction.

### ISSUE THREAD

**Issue by statusquont** [machankura/machankura#2]
I sent some sats to a mobile number of someone who didn't have a Machankura account. But they didn't receive any sort of notice that someone just sent them Bitcoin. If I didn't later tell them through back-channels that 'hey I just sent you some sats' they never would have known, and probably never would have created an account. This seems like a glaringly obvious feature that should be part of the product to help grow the user base.

Here's how I would imagine it would work.

- Existing user sends sats to a phone number
- System checks to see if recipient phone number belongs to a known user account.
  - If new user: push an SMS to that number with a welcome message: "Welcome to Machankura, you just received 12345 satoshis (0.00012345 Bitcoin) from %senders-phone%. To set up your account and claim your money, dial %USSD-code% to get started.
  - If existing user: push an SMS to the number with a funds received message: "you just received 12345 satoshis (0.00012345 Bitcoin) from %senders-phone%, dial %USSD-code% to check your balance."

The backend system would be smart enough to parse the recipients phone number and lookup the country code to insert the correct USSD code.

I realize that there may be some cost to sending these SMS's but hopefully it's cheap enough that your transaction fees would more than cover the cost. Even if it doesn't yet cover the costs at this stage, it seems like it would be a worthwhile business expense to increase user adoption. Perhaps you can cap costs by setting a limit on the Existing Users notifications and make some assumptions that after a user has received X messages you can safely assume they are a regular user and don't need to be prompted any longer. Or perhaps if you have access to metrics about how often a given User dials into the system, you could flag them as a regular user and throttle back SMSs to them. Or you could even make it an option and allow them to choose to enable/disable SMS notifications, and at some point, you could even deduct a couple sats from their balance as a 'pay to receive SMS notifcations feature' from their existing balance.

Long story short, there seems to be a huge missed opportunity to help your existing users increase adoption by more people by letting them market the app to their friends for you (for FREE!) by just using the app.

**Comment by kngako** [comment]
Hello, thank you for the feedback. 

We do send users SMSes notifying them when they are gifted sats. But the gift has to be over 5 001 sats (to offset the cost of the sms). 

Also in some cases the user may just not receieve an SMS (SMSes are a little unreliable at times). So we may have sent the SMS but it may have just not been delivered (or only gets delivered way later then expected).  We will implement a "reminder" SMS, (only for amounts above 10 000 sats though).

---

That being said we believe it is also important that you are in active conversation with the person you are sending sats to and tell them that you have sent them sats (to help them thinking that the SMS they will receive from us is spam)

**Comment by statusquont** [comment]
Ah okay, that must have been what happened, I sent an amount smaller than 5,001. I've just trying to send small teaser amounts to get people interested to dive deeper into learning once they have a little skin in the game. But at the same time it would be a shame to waste sats on someone that isn't going to appreciate them or put in the work to learn, so I was being a bit stingy to preserve my precious sats. ;)
